Sources/Hoa_MultiEncoder.hpp

```diff
@@ -172,6 +172,11 @@ namespace hoa
             
             for(size_t i = 1; i < nencoders; i++)
             {
+                if(m_encoders[i].getMute())
+                {
+                    continue;
+                }
+
                 m_encoders[i].process(++input, m_temp);
```

Comment:
It seems to me that the way the `input` pointer is incremented can cause trouble here with the code you added. 
If you have 3 sources, and the second is muted, I think that the third one will have the signal corresponding to the second source and not the third one, can you confirm this?

Why not access the `input` buffer by index only if the source isn't muted?

*e.g.*:

```cpp
if(!getMute(i))
{
    m_encoders[i].process(input[i], m_temp);
    sigadd(nharmos, m_temp, outputs);
}
```

Reply:
Yes, you absolutely right, your version is correct!

```cpp
if(!getMute(i))
{
    m_encoders[i].process(input[i], m_temp);
    sigadd(nharmos, m_temp, outputs);
}
```